helm/prometheus-rules/templates/alerting-rules/deployment.management-cluster.rules.yml

```diff
@@ -184,7 +170,7 @@ spec:
       annotations:
         description: '{{`Deployment {{ $labels.namespace}}/{{ $labels.deployment }} is not satisfied.`}}'
         opsrecipe: deployment-not-satisfied/
-      expr: kube_deployment_status_replicas_unavailable{cluster_type="management_cluster", deployment=~"worker-.+|master-.+"} > 0
+      expr: kube_deployment_status_replicas_unavailable{cluster_type="management_cluster", deployment=~"worker-.+|master-.+|kyverno.*"} > 0
```

Comment:
Why was Kyverno added here?

Reply:
it was the only deployment in `DeploymentNotSatisfiedLudacris` which I killed - needed to go into either phoenix or rocket, so i chose rocket cause you're closer to that team (we can move it to zachurity in future)

Reply:
Okay - it should go with me to `honeybadger` for now. We use it for releases in CAPI